lib/core/src/lib/card-view/components/card-view/card-view.component.scss

```diff
@@ -1,28 +1,84 @@
-/* stylelint-disable no-descending-specificity */
 .adf-property-list {
     background: var(--adf-card-view-background);
     border: var(--adf-card-view-border);
     border-color: var(--adf-card-view-border-color);
     border-radius: var(--adf-card-view-border-radius);
 
-    .adf-property {
-        margin-bottom: 20px;
-
-        .adf-property-value-padding-top {
-            margin-top: 6px;
-        }
+    .adf-property-label {
+        color: var(--adf-metadata-property-panel-text-color);
+        display: flex;
+        padding: 6px 0;
+        line-height: 20px;
+    }
 
+    .adf-property {
         .adf-property-field {
             width: 100%;
-            margin-bottom: -25px;
+
+            .mat-form-field-underline {
+                display: none;
+            }
+
+            .mat-input-element {
+                text-overflow: ellipsis;
+                color: var(--adf-metadata-property-panel-title-color);
+                margin-top: 32px;
+                padding: 6px 0;
+                line-height: 20px;
+            }
 
             .mat-form-field-infix {
```

Comment:
can we avoid styling mat classes?

Reply:
we have separate https://alfresco.atlassian.net/browse/ACS-6140 to address all mat styling